x/collection/msgs.go

```diff
@@ -221,6 +222,16 @@ func validateChange(change Attribute, validators map[string]func(string) error)
 	return validator(change.Value)
 }
 
+func canonicalKey(key string) string {
+	convert := map[string]string{
```

Comment:
Is the use of a map rather than a simple if statement to make it easier to respond when another change occurs later? For example if `Meta` is deprecated and `Metadata` should be used then just add `AttributeKeyMeta.String(): AttributeMetadata.String(),` ?

Reply:
Exactly.